ocs_ci/ocs/dr/dr_workload.py

```diff
@@ -138,3 +142,82 @@ def delete_workload(self):
 
         config.switch_to_cluster_by_name(primary_cluster_name)
         dr_helpers.wait_for_workload_resource_deletion(self.workload_namespace)
+
+    def data_integrity_check(self, workload_namespace=None, workload_pattern=None):
```

Comment:
If this method is common for all workloads (busybox and mysql), maybe we can move it to base class.